### DISCUSSION TITLE
Genius Romanized

### DISCUSSION THREAD

**Discussion by KosmicAnomaly** [johnwmillr/LyricsGenius#213] [Q&A]
What is the best way to retrieve the romanized lyrics from genius given a song title? I tried using

`genius.search_song("TOMORROW X TOGETHER - 0X1=LOVESONG (I Know I Love You) (Romanized)", "TOMORROW X TOGETHER")`

hoping to retrieve the lyrics to the song with [this url](https://genius.com/Genius-romanizations-tomorrow-x-together-0x1-lovesong-i-know-i-love-you-romanized-lyrics), but the search returned no results.

**Comment by allerter** [comment]
The reason nothing comes up is that you've supplied the wrong artist. The artist for that song is actually "Genius Romanizations".
```
song = genius.search_song(
    "TOMORROW X TOGETHER - 0X1=LOVESONG (I Know I Love You) (Romanized)",
    "Genius Romanizations")
```
